ImageD11/sinograms/dataset.py

```diff
@@ -144,6 +207,8 @@ def import_all(self, scans=None, shape=None):
         
     def import_scans(self, scans=None, hname = None):
         """ Reads in the scans from the bliss master file """
+        # we need to work out what detector we have at this point
+        self.guess_detector()
```

Comment:
Dot we need one detector per scan? How does it work if someone does dct then far field in the same master file